docs/DeveloperGuide.md

```diff
@@ -836,9 +882,20 @@ testers are expected to do more *exploratory* testing.
 
     1. Test case: `export data/MyEmailsList`<br>
        Expected: Exports current listed persons' emails to `MyEmailsList.txt` in the `data` directory.
+   
     1. Test case: `find g/f` followed by `export female_emails`<br>
          Expected: Exports only the emails of female persons.
 
+### Clearing listed participants : `clear` then `confirm`
+
+1. Clears the listed persons from the address book.
+
+    1. Test case: `list` followed by `clear` then `confirm`<br>
+       Expected: Clears the entire address book.
+   
+    1. Test case: `find g/f` followed by `clear` then `confirm`<br>
```

Comment:
Perhaps this should be 2. instead of 1.? XD same issue at line 886!

Reply:
all of them are 1. for some reason idk why... i just followed the rest lol